stl/inc/__msvc_int128.hpp

```diff
@@ -1367,6 +1393,20 @@ struct _Signed128 : _Base128 {
         _Word[1] |= _That._Word[1];
         return *this;
     }
+
+    _TEMPLATE_CLASS_INTEGRAL(_Ty)
+    _NODISCARD_FRIEND constexpr _Ty operator>>(_Ty _Left, const _Signed128& _Right) noexcept {
+        return static_cast<_Ty>(_Signed128{_Left} >> _Right);
+    }
+
+    _NODISCARD constexpr explicit operator bool() const noexcept {
+        return _Word[0] != 0 || _Word[1] != 0;
+    }
+
+    //for static_cast<double>
+    _NODISCARD constexpr explicit operator double() const noexcept {
+        return static_cast<double>(_Word[1]) * 18446744073709551616.0 + static_cast<double>(_Word[0]);
+    }

```

Comment:
This seems wrong for negative values.

Reply:
fixed at [`636907b` (#3559)](https://github.com/microsoft/STL/pull/3559/commits/636907b6f17a20b015e53add248f36907b3cbeb4)